deploy/helm/templates/rbac/clusterrole.yaml

```diff
@@ -19,7 +19,16 @@ rules:
 {{- $doInclude := false }}
 {{- range $i, $line := .Files.Lines "config/rbac/role.yaml" }}
   {{- if eq $doInclude true }}
-{{- $line | nindent 2 }}
+    {{- $line | nindent 2 }}
   {{- end }}
   {{- if eq $line "rules:" }}{{- $doInclude = true }}{{- end }}
 {{- end }}
+{{- if eq .Values.rbac.enabled  true }}
```

Comment:
this part is wrong, what we need is a separate clusterrole object, instead of merge them into manager-role object